core/src/main/java/org/apache/druid/math/expr/Evals.java

```diff
@@ -71,4 +71,14 @@ public static boolean asBoolean(@Nullable String x)
   {
     return !NullHandling.isNullOrEquivalent(x) && Boolean.parseBoolean(x);
   }
+
+  public static long doubleToLongBits(double x)
+  {
+    return Double.doubleToLongBits(x);
```

Comment:
Why add this instead of calling the thing in `Double`?

Reply:
Trying to remember.. I think i was just trying to be consistent with the other value conversion functions and put it here, but it probably could just call it directly as it seems unlike we would change the function